rspec-cells.gemspec

```diff
@@ -7,10 +7,19 @@ Gem::Specification.new do |s|
   s.platform    = Gem::Platform::RUBY
   s.authors     = ["Nick Sutterer"]
   s.email       = ["apotonick@gmail.com"]
-  s.homepage    = "http://rubygems.org/gems/rspec-cells"
+  s.homepage    = "https://rubygems.org/gems/rspec-cells/versions/#{s.version}"
```

Comment:
should be this github page.

Reply:
Updated in 7238982d4ef8faf618c3925ef91fea43e42acfbf